src/booking/containers/booking-date/booking-date.component.scss

```diff
@@ -0,0 +1,6 @@
+/deep/.date-head h1 {
+  margin-top: 0px;
+  margin-bottom: 10px;
+  font-weight: bold !important;
```

Comment:
@tarikdogan-hmcts - I would try to avoid using the `!important` override. If it's really essential, then consider doing it the GDS way by using the `govuk-!-font-weight-bold` CSS class (see https://design-system.service.gov.uk/styles/typography/). @paulhowes-HMCTS - what do you reckon?

Reply:
Marking as resolved, given that this CSS rule is no longer required.